lib/rules/web/admin_console/4.3/check_links.xyaml

```diff
@@ -56,3 +56,10 @@ navigate_to_dev_console:
   - selector:
       xpath: //li//h1[contains(., 'Developer')]
     op: click
+check_external_link:
+  elements:
+  - selector:
+      xpath: //h2[text()='Launcher']
+    timeout: 20
+  - selector:
+      xpath: //a[contains(@class,'co-external-link') and text()='<link_text>']
```

Comment:
does this rule can be used here?
`check_link_and_text:
  element:
    selector:
      xpath: //a[contains(.,'<text>') and contains(@href,'<link_url>')]`

Reply:
This rule seems not applicable for this scenario since all links have the same <link_url>. They just have different text

Reply:
Ok, got it.